sidebars.js

```diff
@@ -540,6 +540,7 @@ const sidebars = {
       },
       'items': [
         'deployment/macOS',
+        'deployment/debian_ubuntu',
```

Comment:
Should rename this to `debian-ubuntu` (and the md file itself) so that it's consistent with the rest of the md files that use `kebab-case`.

Reply:
Done